src/main/java/nl/tudelft/ti2206/group9/ShaftEscape.java

```diff
@@ -0,0 +1,81 @@
+package nl.tudelft.ti2206.group9;
+
+import javafx.application.Application;
+import javafx.stage.Popup;
+import javafx.stage.Stage;
+import nl.tudelft.ti2206.group9.gui.AbstractScene;
+import nl.tudelft.ti2206.group9.gui.SplashScene;
+import nl.tudelft.ti2206.group9.util.GameObservable;
+import nl.tudelft.ti2206.group9.util.Logger;
+
+/**
+ * Starting point of the Application.
+ * Contains method to switch Scenes or show Popups.
+ * @author Maarten
+ */
+@SuppressWarnings("restriction")
+public class ShaftEscape extends Application {
+
+	private static Stage stage;
+	/** Width of the Window. */
+	public static final int WIDTH = 480 - 16;
```

Comment:
You should try it yourself, but apparently without the `-16`, the background is too small for the window. There should be a better way to fix this, but this also does the trick.
<review